src/screens/About/About.tsx

```diff
@@ -75,7 +75,7 @@ const About = ({ children }: { children: React.ReactNode }) => {
           {content.partnersContent.map((section: any, i: number) => {
             return (
               <Fragment>
-                <BodyCopy source={section.copy} linkTarget="_blank" />
+                <BodyCopy key={i} source={section.copy} linkTarget="_blank" />
```

Comment:
you will need to put the key in the `Fragment` element, because that's the one repeating under `Wrapper`